guides/source/configuring.md

```diff
@@ -549,7 +549,7 @@ Defaults to `'signed cookie'`.
 
 * `config.action_view.logger` accepts a logger conforming to the interface of Log4r or the default Ruby Logger class, which is then used to log information from Action View. Set to `nil` to disable logging.
 
-* `config.action_view.erb_trim_mode` gives the trim mode to be used by ERB. It defaults to `'-'`, which turns on trimming of tail spaces and newline when using `<%= -%>` or `<%= =%>`. See the [Erubis documentation](http://www.kuwata-lab.com/erubis/users-guide.06.html#topics-trimspaces) for more information.
```

Comment:
I kept this on purpose since erubi doesn't have documentation and the documentation is the same. Could you revert this change?

Reply:
Reverted